src/include/homestore/index/wb_cache_base.hpp

```diff
@@ -73,7 +73,7 @@ class IndexWBCacheBase {
     /// @brief Copy buffer
```

Comment:
Can we update the API comment for `create_chain`, especially the @brief part, which doesn't seem correct to me.